docs/development/dapr-metrics.md

```diff
@@ -54,6 +55,15 @@ Dapr uses prometheus process and go collectors by default.
 * dapr_sentry_issuercert_changed_total: The number of issuer cert updates, when issuer cert or key is changed
 * dapr_sentry_issuercert_expiry_timestamp: The unix timestamp, in seconds, when issuer/root cert will expire.
 
+## Dapr Scheduler metrics
+
+[monitoring metrics](../../pkg/scheduler/monitoring/metrics.go)
+
+* dapr_scheduler_sidecars_connected_total: The total number of dapr sidecars connected to the scheduler service.
+* dapr_scheduler_jobs_created_total: The total number of jobs scheduled.
+* dapr_scheduler_jobs_triggered_total: The total number of successfully triggered jobs.
```

Comment:
How will this behave once we have added failure policy- I guess we will need to add labels for `SUCCESS`, `FAILED`, and `UNDELIVERED`.

`UNDELIVERED` will be tricky as we might need to track when they do get success/failure triggered so this gauge can decrease.

Reply:
I think adding labels will be the easiest. Since that change is not in, Id like to not wait or add a pre-req to this PR since its been open a while already.